recipes/autoconf/all/conanfile.py

```diff
@@ -1,8 +1,13 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-import contextlib
-import os
+from os import environ
+from pathlib import Path
 
-required_conan_version = ">=1.33.0"
+from conan import ConanFile
+from conan.tools.files import get, AutoPackager, apply_conandata_patches, rmdir
+from conan.tools.gnu import Autotools
+from conan.tools.layout import basic_layout
+from conans.client.subsystems import deduce_subsystem, subsystem_path
```

Comment:
It can not be import, it's private import. Besides that, subsystem should be managed from conf: https://docs.conan.io/en/latest/migrating_to_2.0/recipes.html#windows-subsystems

Reply:
But unfortunately it is not see https://github.com/conan-io/conan/issues/11975

I don't see an immediate way around fixing that issue in conan or adding a custom function to the recipe which transform those paths

Reply:
Not using the private imports results in the following error on windows with msys2 and using Conan dev branch with the following fix enabled https://github.com/conan-io/conan/pull/11981

```powershell
autoconf/2.71 (test package): Calling build()
autoconf/2.71 (test package): run_in_windows_bash: C:\.conan\7c3916\1\bin\msys64\usr\bin\bash.exe --login -c ^"cd \^"/c/cura_wp/conan-center-index/recipes/autoconf/all/test_package/build/373a667231cd5dd4f20398c7d2cb2f5ccdb95ce6\^" ^&^& PATH=\^"/c/.conan/f2141b/1/bin:/c/.conan/7c3916/1/bin/msys64/usr/bin:/c/.conan/7c3916/1/bin:/c/.conan/bc0586/1/bin:/c/.conan/bc0586/1/bin:$PATH\^" ^&^& C:\.conan\f2141b\1\bin\autoconf --verbose ^"

----Running------
> C:\.conan\7c3916\1\bin\msys64\usr\bin\bash.exe --login -c ^"cd \^"/c/cura_wp/conan-center-index/recipes/autoconf/all/test_package/build/373a667231cd5dd4f20398c7d2cb2f5ccdb95ce6\^" ^&^& PATH=\^"/c/.conan/f2141b/1/bin:/c/.conan/7c3916/1/bin/msys64/usr/bin:/c/.conan/7c3916/1/bin:/c/.conan/bc0586/1/bin:/c/.conan/bc0586/1/bin:$PATH\^" ^&^& C:\.conan\f2141b\1\bin\autoconf --verbose ^"
-----------------
/usr/bin/bash: line 1: C:.conanf2141b1binautoconf: command not found
ERROR: autoconf/2.71 (test package): Error in build() method, line 36
        self.run("{} --verbose".format(os.environ["AUTOCONF"]),
        ConanException: Error 127 while executing C:\.conan\f2141b\1\bin\autoconf --verbose
```

Reply:
Thx for pointing auto the list of linter errors in my [qt] PR just came across this https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html#conan-tools-microsoft-unix-path

That should take car of the hidden imports

Reply:
should be fixed now